packages/@okta/vuepress-site/docs/guides/custom-widget/main/index.md

```diff
@@ -413,9 +454,9 @@ For a more in-depth look at styling the sign-in page, you can watch this video:
 
 ### Modify strings
 
-To modify strings on the sign-in page, you can override any of the properties set in [login.properties](https://github.com/okta/okta-signin-widget/blob/master/packages/@okta/i18n/src/properties/login.properties). You override these properties by specifying new values for them inside an `i18n` object in the `config` section of the sign-in page.
+To modify strings on the sign-in page, you can override any of the properties set in [login.properties](https://github.com/okta/okta-signin-widget/blob/master/packages/@okta/i18n/src/properties/login.properties). Override these properties by adding new values for them. Configure the `i18n` object in the `config` section of the sign-in page.
 
-You can modify any of the labels found on the sign-in page by providing new values for them.
+You can add new values for any of the labels found on the sign-in page.
```

Comment:
You can use "You can add values for..."; if you add something, the 'new' is implied.

Reply:
thank you.